### DISCUSSION TITLE
table设置scroll-y，当表格没有值，当前列的表头长度不会根据本身的长度固定，如何设置表头文字不换行

### DISCUSSION THREAD

**Discussion by z-lionel** [ant-design/ant-design#33229] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/table-cn/](https://ant.design/components/table-cn/)

### Steps to reproduce
table组件设置scroll，表头名字长点

### What is expected?
表头不换行

### What is actually happening?
表头文字换行了

| Environment | Info |
|---|---|
| antd | undefined |
| React | reacet |
| System | window |
| Browser | chrome |

---
table设置scroll:{y: 1800}的时候，当列没有内容，或者比列名短的时候，列名会自动换行，请问有不换行的配置吗？还是需要自定义样式处理？

**Comment by JackDan9** [comment]
我一般回去动态获取table的宽度，就是通过useRef去获取回来当前的宽度和高度，然后放到srcoll中，因为设置了`scroll`为`y:true`或者`y: 'max-content'` `fixed`和`width`又会被覆盖掉(当然自定义样式也会解决这个问题)，所以几种办法都是可以的。
